File: inform/1/f.py
n = int(input())
j = 0
sum = 0
l = 0
while (sum < n):
    j += 1
    sum += j
    l += 1

i = 0
while (sum != n):
    if (sum > n):
        i += 1
        sum -= i
        l -= 1
    else:
        j += 1
        sum += j
        l += 1
print(l)

File: inform/1/f.cpp
#include <iostream>
#include <vector>
using namespace std;
using ll = long long;

const int INF = 1e9 + 7;

int main() {
    ll n;
    cin >> n;
    ll j = 0;
    ll sum = 0;
    ll len = 0;
    while (sum < n) {
        j++;
        sum += j;
        len++;
    }
    // cout << sum << " " << len << endl;
    ll i = 0;
    while (sum != n) {
        if (sum > n) {
            i++;
            sum -= i;
            len--;
        } else {
            j++;
            sum += j;
            len++;
        }
    }
    cout << len << endl;
    // cout << sum << " " << len << " " << i << " " << j << endl;
    return 0;
}